01-Login/README.md

```diff
@@ -68,4 +75,1059 @@ If you have found a bug or if you have a feature request, please report them at
 
 ## License
 
-This project is licensed under the MIT license. See the [LICENSE](LICENSE.txt) file for more info.
+This project is licensed under the MIT license. See the [LICENSE](../LICENSE) file for more info.
+
+# Tutorial
+This tutorial will help you get up and running the React app using the new SDK.
+
+## Setting Up the Application
+
```

Comment:
Let's put the steps in here for running `create-react-app` so that it's clear for the reader how to  get started from scratch.

i.e. "Find a location on your drive, run `<npm command here>`, etc.